lizmap/modules/lizmap/lib/Project/configFile.php

```diff
@@ -0,0 +1,389 @@
+<?php
+
+namespace Lizmap\Project;
+
+class configFile
+{
+    protected $data;
+
+    public function __construct($cfgFile)
+    {
+        $this->data = json_decode($cfgFile);
```

Comment:
it seems $cfgFile is a file name. So you should read the content of the file, and give this content to json_decode